EmptyState.jsx

```diff
@@ -0,0 +1,66 @@
+/**
+ * TEAM: frontend_infra
+ * @flow strict
+ */
+import * as React from "react";
+import {StyleSheet, css} from "aphrodite";
+import GraphicIcon from "./GraphicIcon";
+import {whitespaceSizeConstants} from "./styles/whitespace";
+import Text from "./Text";
+import type {GraphicIcons} from "./tools/graphicIcons";
+import typeof Button from "./button/Button";
+
+type Props = {|
+  +icon: GraphicIcons,
+  /** Text for the headline. This should be in sentence case. */
```

Comment:
NIT: consistent capitalization

```suggestion
  /** text for the headline. This should be in sentence case. */
```